snapshotter/snapshotter/manifest.go

```diff
@@ -0,0 +1,174 @@
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package snapshotter
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const MANIFEST_VERSION = "0.2"
+
+//!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+//! The names of these fields must not be changed without fixing virtcontainer.go !!!
+type ImageInfo struct {
+	// The OCI Image Ref of the image
+	ImageRef string
+
+	// The Raw Manifest of the image
+	ManifestRaw []byte
+
+	// The digest of the manifest
+	ManifestDigest string
+
+	// The digest of the image's OCI config
+	ConfigDigest string
+
+	// The RAW bytes of the OCI Config json
+	ConfigRaw []byte
+}
```

Comment:
Is the image info structure contained in the security policy? Or is the whole image manifest contained in the security policy. It looks like including the image info is sufficient since it includes the digest of the image manifest.